transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java

```diff
@@ -499,4 +509,6 @@ protected EventExecutor executor() {
             return GlobalEventExecutor.INSTANCE;
         }
     }
+
+    protected abstract boolean isServerSide();
```

Comment:
This method cannot be abstract because this class is public and it would break the API contract.